web/book/src/tutorial/filtering.md

```diff
@@ -1,74 +1,73 @@
 # Filtering rows
 
-In previous page we learned how `derive` and `select` can used to create new
-columns and remove unwanted ones.
+In previous page we learned how `derive`, `select`, and `join`
+add new columns and remove unwanted ones.
 
-Now we will look into how we can manipulate rows using `filter` and `take`.
+Now we will look into how we can manipulate rows of a table
+using `filter` and `take`.
 
-> Side note: each of the transform functions has at least some invariant: select
-> and derive will not change the number of rows, filter and take will not change
-> the number of columns. We call this separation of effects "transform
-> orthogonality". It's goal is to keep transform functions composable by
-> minimizing interference of their effects.
->
-> In SQL, you can see this lack of invariant when an aggregation function is
-> used in the `SELECT` clause. Before, the number of rows was kept constant, but
-> introduction of an aggregation function caused the whole statement to produce
-> only one row (per group).
-
-Filter pick rows based on their values:
+**filter:** The `filter` transform picks (passes through) rows based on their values:
```

Comment:
```suggestion
**filter:** The `filter` transform picks rows based on their values:
```

I'm not sure `picks` is great either, but `passes through` seems not great, given its role is to not pass things through? "selectively passes through"??

Reply:
I'll change it to say, "... picks the rows to pass through..." in this first description, then use "picks" thereafter. Same for `take`